wp-config.php

```diff
@@ -24,71 +24,9 @@
  */
 else:
   if (isset($_ENV['PANTHEON_ENVIRONMENT'])):
```

Comment:
As long as we're modifying this section, let's switch from the odd `if ():` syntax, and use `{}` as usual here.

Reply:
Already tried that— mixing the two `if` syntaxes did not play nice. We can replace *all* the if / endif throughout the file to use {} if we want, or stay consistent with if/endif— with the increased chance of conflicts (which may be a moot point given we're already expecting some).